index.html

```diff
@@ -16,13 +16,187 @@
+                <tr>
+                  <td>Saturn</td>
+                  <td>1.2 billion km</td>
+                  <td>No</td>
+                  <td>Depends On Orbit (3-6 Years)</td>
+                </tr>
+              </tbody>
+            </table>
       </div>
+      <div class="row mt-5">
+        <div class="col-6">
+          <h1>Let's Go For A Walk</h1>
+          <p>The Earth was small, light blue, and so touchingly alone, our home that must be defended like a holy relic. The Earth was absolutely round.</p>
+          <p>It suddenly struck me that that tiny pea, pretty and blue, was the Earth. I put up my thumb and shut one eye, and my thumb blotted out the planet Earth.</p>
+        </div>
+        <div class="col-6">
+          <img src="./img/space-walk.jpg" alt="Astronaut Walking">
+        </div>
+      </div>
+    <div class="row mt-5">
+      <div class="col-9">
+        <h3>Ready To Go?</h3>
+      </div>
+      <div class="col-3">
+        <h4>Space Stuff</h4>
+      </div>
+      <div class="col-9">
+          <form>
+            <div class="form-group">
+              <label for="name">Namel</label>
+              <input type="text" class="form-control" id="name" placeholder="Name">
+            </div>
+            <div class="form-group">
+              <label for="phone">Phone</label>
+              <input type="number" class="form-control" id="phone" placeholder="Phone">
+            </div>
+            <div class="form-group">
+              <label for="email">Email</label>
+              <input type="email" class="form-control" id="email" placeholder="Email">
+            </div>
+            <div class="form-check">
+                <input class="form-check-input" type="checkbox" value="" id="defaultCheck1">
+                <label class="form-check-label" for="defaultCheck1">
+                    I would like to receive promotional emails
+                </label>
+            </div>
+            <div class="form-check">
+                <input class="form-check-input" type="checkbox" value="" id="defaultCheck1">
+                <label class="form-check-label" for="defaultCheck1">
+                    Please don’t send me emails
+                </label>
+            </div>
+            <button type="submit" class="btn btn-primary mt-1">Ready For Launch</button>
+          </form>
+      </div>
+      <div class="col-3">
+          <ul class="list-group">
+              <li class="list-group-item">Space Helmet</li>
+              <li class="list-group-item">Rocket Boosters</li>
+              <li class="list-group-item">Shuttle</li>
+              <li class="list-group-item">Training for Launch</li>
+              <li class="list-group-item">Heat resistant plating</li>
+              <li class="list-group-item">Space Suit</li>
+            </ul>
+        </div>
+      </div>
+    </div>
+    <div class="jumbotron mt-5">
+        <div class="row footer mt-5">
+            <div class="col-12 justify-content-center">
```

Comment:
Taking a look at the dev tools in google, you can remove the justify-content-center` since it is not being applied here in this case. You may want to use a text align rather than a justify content. We use justify content when dealing with images, and containers. In this case, we are dealing with text!